drivers/include/mma8652.h

```diff
@@ -104,7 +116,7 @@ int mma8652_test(mma8652_t *dev);
  * @return                  -4 if setting to STANDBY mode failed
  * @return                  -5 if accelerometer configuration failed
  */
-int mma8652_init(mma8652_t *dev, i2c_t i2c, uint8_t address, uint8_t dr, uint8_t range);
+int mma8652_init(mma8652_t *dev, i2c_t i2c, uint8_t address, uint8_t dr, uint8_t range, uint8_t type);
```

Comment:
I agree it can be confusing. 
I did not want to do the full API change to avoid breaking compatibility. But we can imagine a couple of macro to translate calls if needed.
So I can still make the change API if you prefer it.